### ISSUE TITLE
Add LEDs

### ISSUE THREAD

**Issue by pglotfel** [robotarium/robotarium_python_simulator#2]
Add LED functionality to the simulator.

**Comment by guyfleeman** [comment]
I'd like work on this feature. Can I assume it will use the same endpoint as the Matlab sim?
Matlab:
function this = set_leds(this, ids, commands)

So a python endpoint would be:
def set_leds(self, ids, commands)

ids is a 1xn numpy array
commands is a 4xN numpy array [r g b led_num]: 0 <= r, g, b <= 255, 0 <= led_num <= 1

Is this correct?

**Comment by pglotfel** [comment]
Yes, it should be the same as the MATLAB simulator; however, we're doing some hardware upgrades very soon.  So I think this issues should probably wait until those upgrades are complete.
